newIDE/app/src/MainFrame/EditorContainers/HomePage/PlaySection/GamesPlatformFrame.js

```diff
@@ -68,6 +71,20 @@ const GamesPlatformFrame = ({ initialGameId, loaded, visible }: Props) => {
     [loaded, initialGameId]
   );
 
+  React.useEffect(
+    () => {
+      if (!iframeRef.current) return;
```

Comment:
I would probably add a check on `!loaded` too.
The iframe may be here without gd.games being loaded yet, or not loaded at all .
This component and the iframe is always in the DOM, but the `src` changes when a user arrives on the Play section, and the `loaded` becomes true when gd.games tells the editor it's ready.